weaviate/values.yaml

```diff
@@ -88,7 +96,14 @@ securityContext: {}
 
 # Security context for the Weaviate container. Override overlapping settings made at the Pod level.
 containerSecurityContext: {}
-
+  # runAsUser: 1000
+  # runAsGroup: 1000
+  # fsGroup: 1000
+  # fsGroupChangePolicy: "OnRootMismatch"
```

Comment:
The only subproperties from containerSecurityContext I see being used are runAsUser and fsGroup, why do we expose the rest on the values.yml (even if commented) if they won't be applied in the STS? or were they missed?